### ISSUE TITLE
Screen reader is not reading all content present in the pdf after exporting to pdf.

### ISSUE THREAD

**Issue by msftedad** [adobe/adobe.github.com#143]
#### Expected Behaviour
Screen reader should read all content present in the pdf after exporting to pdf.
#### Actual Behaviour
Screen reader is not reading all content present in the pdf after exporting to pdf.

#### Steps to Reproduce
1. Open any saved PDF file in Adobe, Turn on the screen reader.
2. Navigate through the PDF file in the Adobe reader.
3. Observe the issue. 
 
#### Platform and Version
OS: Windows 11 Version 24H2 (OS Build 26100.1)
Tool Used: Adobe Acrobat 24.002.20687

[Project Planning (1) (2).pdf](https://github.com/adobe/adobe.github.com/files/15130712/Project.Planning.1.2.pdf)
![Narrator (1)](https://github.com/adobe/adobe.github.com/assets/93735775/1de4f47b-20a3-4c26-92d7-287f241cd959)
![NVDA (1)](https://github.com/adobe/adobe.github.com/assets/93735775/bbe44f86-d65b-4fb7-9e21-28fc96358dec)

**Comment by lesha0** [comment]
Hey @msftedad,

Seems like it might be an accessibility issue in your PDF. In Adobe Acrobat, try activating "Read Out Loud" from the View menu. 

If that doesn't work, check how you're making the PDF. Real text, proper ALT tags for images, and structured tables can help screen readers work better. 

Review the accessibility of your output PDF using the Accessibility Checker in Adobe Acrobat. This can spot any problems.

Make sure to check whether your PDF generator supports creating accessible PDFs. 

Hope this helps! Let me know how it goes.

**Comment by msftedad** [comment]
@lesha0 , Thanks for sharing the information. We will check this with our internal team and if any further queries we will get back to you.

**Comment by msftedad** [comment]
Hi @lesha0 , In Adobe acrobat, we have tried turning on the Read Aloud, where it announces as blank when the focus goes to link. 

But we are unable to spot the issue with the Accessibility Checker and also, PDF generator supports creating accessible PDFs.

https://github.com/user-attachments/assets/fdc86c48-6945-402d-b736-33dda6be9160

**Comment by msftedad** [comment]
Hi @lesha0 , Any update?

**Comment by msftedad** [comment]
Hi Team, Any update on this?

**Comment by msftedad** [comment]
Hi Team, Any update on this?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?
